.eslintrc

```diff
@@ -28,13 +27,16 @@
         "allowBlockStart": false
       }
     ],
+    "babel/no-unused-expressions": "off",
     "import/named": "off",
     "import/no-named-as-default": "off",
     "react/button-has-type": "off",
     "react/no-array-index-key": "off",
     "react/jsx-fragments": ["error", "element"],
     "shopify/jsx-no-complex-expressions": "off",
+    "shopify/jsx-prefer-fragment-wrappers": "off",
     "shopify/no-ancestor-directory-import": "error",
+    "shopify/react-prefer-private-members": "off",
```

Comment:
Sometimes we do actually want pubic functions

Reply:
I wonder if we should enable this and add an ignore whenever public is used, which is only in `Banner` i think?

Reply:
There's 36 violations currently, 1/3 of which are false positives warning about examples written in JS when you can't even use private members (bug raised: https://github.com/Shopify/eslint-plugin-shopify/issues/259), and many of the others seem related to legacy context stuff - subscribe/unsubscribe stuff. I don't know enough about the code they touch to know what should be private or not.

I'd totally be open to reevaluating this in the future, but I don't want to touch them now :)